### PR TITLE
[chrome.angle] Force Pure RISCV64 Support

### PR DESCRIPTION
**Pull request by McKnight22**

### CODE REVIEW

**Comment by unicornx** [comment]
I changed the tile of this PR to keep align with other PRs for chrome porting work, please check it out.

File: android/angle_apk.gni [hunk 1]
```diff
@@ -46,14 +46,14 @@ if (enable_java_templates) {
       deps = [ ":${invoker.package_name}_assets" ]
       if (symbol_level != 0) {
         deps += [ ":compressed_symbols" ]
-        if (android_64bit_target_cpu) {
+        if (android_64bit_target_cpu && !invoker.skip_secondary_abi) {
```
Comment (unicornx) [inline android/angle_apk.gni]: Please clarify why you find out these two changes.

Comment (McKnight22) [reply android/angle_apk.gni]: > Please clarify why you find out these two changes.

For these 2 changes, it's due to android_secondary_abi_toolchain introduced here and android_secondary_abi_toolchain should be skipped for Android RISCV64 target to enable pure RISCV64 support.

File: android/angle_apk.gni [hunk 2]
```diff
@@ -46,14 +46,14 @@ if (enable_java_templates) {
       deps = [ ":${invoker.package_name}_assets" ]
       if (symbol_level != 0) {
         deps += [ ":compressed_symbols" ]
-        if (android_64bit_target_cpu) {
+        if (android_64bit_target_cpu && defined(android_secondary_abi_cpu)) {
```
Comment (unicornx) [inline android/angle_apk.gni:49]: 2nd review:
建议用 `android_secondary_abi_toolchain`, 和下面代码结合比较紧密，好理解。

File: android/angle_apk.gni [hunk 3]
```diff
@@ -46,14 +46,14 @@ if (enable_java_templates) {
...
           deps += [ ":compressed_symbols($android_secondary_abi_toolchain)" ]
         }
       }
 
       uncompress_shared_libraries = true
 
-      if (android_64bit_target_cpu) {
+      if (android_64bit_target_cpu && defined(android_secondary_abi_cpu)) {
```
Comment (unicornx) [inline android/angle_apk.gni:56]: 2nd review:
建议用 `android_secondary_abi_toolchain`, 和下面代码结合比较紧密，好理解。

**Comment by unicornx** [comment]
先关闭这个 pr，新建 https://github.com/aosp-riscv/angle/pull/5 作为总结性提交。

**Pull request closed without merging**